tfhe/src/high_level_api/booleans/base.rs

```diff
@@ -48,6 +50,19 @@ impl Named for FheBool {
     const NAME: &'static str = "high_level_api::FheBool";
 }
 
+impl ParameterSetConformant for FheBool {
+    type ParameterSet = RadixCiphertextConformanceParams;
+
+    fn is_conformant(&self, params: &RadixCiphertextConformanceParams) -> bool {
+        assert_eq!(params.num_blocks_per_integer, 1);
```

Comment:
shouldn't this return false rather than panic ?

Reply:
ah no I see, you need a specific set of conformant parameter, how about an intermediate type that a RadixCiphertextConformanceParams can be converted into hardcoding the num blocks to 1 ?

Reply:
This is so we can keep the same entry point https://github.com/zama-ai/tfhe-rs/blob/ebce4fcfd4d4c9a3dfeaa10a7c39501b0c44198e/tfhe/src/high_level_api/integers/mod.rs#L119

There's the plain CiphertextConformantParameter  but due to the bounds on the function above, it woud mean either we create a different function or somehow change the bounds.

Otherwise, we could change the panic with an error message if the len is not one saying FheBool only expect one block

Reply:
or we hard code the number of blocks here to 1 to check the number of blocks in the boolean blcok directly ? and have a todo to fix later

Reply:
is conformant is not supposed to crash normally it's a safe way to check the data is malformed or well formed

Reply:
remember also that this is a method against adversarial data so we need to check the number of blocks and cannot rely on the HL API constant